src/plugins/intel_gpu/src/graph/impls/common/loop.cpp

```diff
@@ -37,63 +107,173 @@ struct loop_impl : typed_primitive_impl<loop> {
     void set_node_params(const program_node& arg) override {
         OPENVINO_ASSERT(arg.is_type<loop>());
         const auto& node = arg.as<loop>();
-        _max_iteration = node.get_max_iteration();
         _back_edges = node.get_back_edges();
     }
 
+    void set_body_output_memory(cldnn::network::ptr network, const std::shared_ptr<cldnn::primitive_inst>& inst, memory::ptr mem) const {
+        if (inst->is_input()) {
+            network->set_input_data(inst->id(), mem);
+        } else if (inst->is_output()) {
+            network->set_output_memory(inst->id(), mem);
+        } else {
+            inst->set_output_memory(mem, false);
+        }
+    }
+
+    void change_memory_buffer_static_model(const loop_inst::backedge_memory_mapping& mapping, network::ptr network, int64_t iter) const {
```

Comment:
"change buffer " is also not very exact for its purposed, likely it seems "handle_buffers_for_next_iteration" ?

Reply:
Fixed it. thanks.